osgbuild/main.py

```diff
@@ -570,12 +585,13 @@ def parser_targetopts_callback(option, opt_str, value, parser, *args, **kwargs):
         assert kojiinter  # shouldn't get here without kojiinter
         for dver in targetopts_by_dver:
             targetopts_by_dver[dver]['koji_tag'] = 'TARGET'
-    elif opt_str == '--upcoming':
+    elif opt_str.endswith('upcoming'):
```

Comment:
`--coffeecupcoming` !

Reply:
I appreciate the sentiment -- I'd like to have some coffee coming up :)
`--coffeeupcoming` wouldn't get this far -- optparse would reject it instead of sending it to the callback.